mobile/src/navigations/meditation/meditation.tsx

```diff
@@ -0,0 +1,31 @@
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
```

Comment:
import React from 'react'; always goes on first line

Reply:
No) don't forget that the `npm run lint:js -w mobile -- --fix` command automatically sorts imports